kolibri/deployment/default/settings/base.py

```diff
@@ -47,8 +47,10 @@
     'kolibri.auth.apps.KolibriAuthConfig',
     'kolibri.content',
     'kolibri.logger',
+    'kolibri.tasks.apps.KolibriTasksConfig',
```

Comment:
Oh, interesting. Seems this explicit method of specifying the AppConfig is the new recommended practice: https://docs.djangoproject.com/en/1.9/ref/applications/#configuring-applications